spec/factories/consent_responses.rb

```diff
@@ -39,22 +40,6 @@
 #  fk_rails_...  (campaign_id => campaigns.id)
 #  fk_rails_...  (patient_id => patients.id)
 #
-HEALTH_QUESTIONS = {
-  flu: [
-    "Does the child have a disease or treatment that severely affects their immune system?",
-    "Is anyone in your household having treatment that severely affects their immune system?",
-    "Has your child been diagnosed with asthma?",
-    "Has your child been admitted to intensive care because of a severe egg allergy?",
-    "Is there anything else we should know?"
-  ],
-  hpv: [
-    "Does the child have any severe allergies that have led to an anaphylactic reaction?",
-    "Does the child have any existing medical conditions?",
-    "Does the child take any regular medication?",
-    "Is there anything else we should know?"
-  ]
-}.freeze
-
```

Comment:
I think the reason these made sense here was that we think we'll eventually have to put these into the database in their own table. I'm not sure it matters much if they're here or in `ConsentResponse`.